latch_cli/snakemake/config/utils.py

```diff
@@ -148,11 +166,14 @@ def parse_value(t: Type, v: JSONValue):
         return v, v
 
     if isinstance(v, list):
-        assert get_origin(t) is list
+        assert get_origin(t) is list or get_origin(t) is Annotated
```

Comment:
would actually put this at the top

```
def parse_value(t: Type, v: JSONValue):
    if get_origin(t) is Annotated:
        args = get_args(t)
        assert len(args) > 0
        
        return parse_value(args[0], v)
```

this way we support any valid annotated type